index.js

```diff
@@ -9,5 +11,30 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     forceHighlander(this.project);
+
+    let app = this._findHost();
+    if (!app.tests) {
+      let checker = new VersionChecker(this.parent).for('ember-cli-babel', 'npm');
+
+      app.options = app.options || {};
+      if (checker.isAbove('7.0.0')) {
+        app.options.babel = app.options.babel || {};
+        app.options.babel.plugins = app.options.babel.plugins || [];
+        app.options.babel.plugins.unshift([StripWrappers, { legacy: true }]);
+      }
+    }
+  },
+
+  _findHost() {
```

Comment:
Because it's a private method, so and I didn't want to create a dependency on the interface to a private method, possibly causing a dependency on a specific minor/bugfix version of `ember-cli` in the future if that private method's signature is ever changed.